views/deck.handlebars

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+    <div class="row">
+        <div class="col-8 offset-2">
+            <div class="card mt-4">
```

Comment:
the mt-4 margin property is not available in materilize, we can delete that class